reporters/GlobalSummaryReporter.js

```diff
@@ -0,0 +1,237 @@
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.collateralReturned));
+      }
+    }
+    for (let event of liquidationRewardEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.withdrawalAmount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.withdrawalAmount));
+      }
+    }
+    for (let event of regularFeeEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.regularFee)).add(toBN(event.lateFee));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.regularFee)).add(toBN(event.lateFee));
+      }
+    }
+    for (let event of finalFeeEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.amount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.amount));
+      }
+    }
+
+    allSponsorStatsTable["collateral withdrawn"] = {
+      cumulative: this.formatDecimalString(collateralWithdrawn),
+      "24H": this.formatDecimalString(collateralWithdrawnDaily)
+    };
+
+    // - Net collateral deposited into contract:
+    let netCollateralWithdrawn = collateralDeposited.sub(collateralWithdrawn);
+    let netCollateralWithdrawnDaily = collateralDepositedDaily.sub(collateralWithdrawnDaily);
+    allSponsorStatsTable["net collateral deposited"] = {
+      cumulative: this.formatDecimalString(netCollateralWithdrawn),
+      "24H": this.formatDecimalString(netCollateralWithdrawnDaily)
+    };
+
+    // - Tokens minted: Creates
+    let tokensMinted = toBN("0");
+    let tokensMintedDaily = toBN("0");
+    for (let event of createEvents) {
+      tokensMinted = tokensMinted.add(toBN(event.tokenAmount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        tokensMintedDaily = tokensMintedDaily.add(toBN(event.collateralAmount));
+      }
+    }
+    allSponsorStatsTable["tokens minted"] = {
+      cumulative: this.formatDecimalString(tokensMinted),
+      "24H": this.formatDecimalString(tokensMintedDaily),
+      current: this.formatDecimalString(await this.empContract.methods.totalTokensOutstanding().call())
+    };
+
+    // - Tokens repaid: Redeems, SettleExpired's
+    let tokensRepaid = toBN("0");
+    let tokensRepaidDaily = toBN("0");
+    for (let event of redeemEvents) {
+      tokensRepaid = tokensRepaid.add(toBN(event.tokenAmount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        tokensRepaidDaily = tokensRepaidDaily.add(toBN(event.collateralAmount));
+      }
+    }
+    for (let event of expirySettlementEvents) {
+      tokensRepaid = tokensRepaid.add(toBN(event.tokensBurned));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        tokensRepaidDaily = tokensRepaidDaily.add(toBN(event.tokensBurned));
+      }
```

Comment:
High level question: should we define tokens as "repaid" when the liquidation is initiated since that's when they're technically burned? Or should we wait until the liquidation is settled?

Reply:
Oh that's a good point. Two things: 
- (1) it's probably clearer to change language "repaid" to "burned" since it counts all tokens that are destroyed. 
- (2) My general philosophy in this report has been to report only token amounts that go in/out of the contract. So a liquidation would count as "tokens burned" since tokens are destroyed

@allisonlu22 what do you think?